examples/4_mine_and_craft.py

```diff
@@ -7,7 +7,7 @@
 from tagilmo.utils.mathutils import normAngle, degree2rad
 
 import numpy as np
-from minelogy import get_log_names, initialize_minelogy
+from minelogy import minelogy
```

Comment:
class name might be better to capitalize (`Minelogy`)

Reply:
> class name might be better to capitalize (`Minelogy`)

Done